### ISSUE TITLE
Custom Form.io events not being triggered and pushed through vue

### ISSUE THREAD

**Issue by GabrielGoldsteinAnidea** [formio/vue#52]
I have a JSON form generated by an embedded device which I then load into a vue component.  I create the form and later inject the JSON into :form.  This all works well.  I have some buttons in the form which trigger a custom event, which I've found no way to get to trigger.  

Vue template:
`
<formio :form="this.roomOverviewHtml" :options="this.formioOptions" v-on:submit="this.myFormsubmit" v-on:customEvent="this.postButton" v-on:requestButton="this.postButton" v-on:submitButton="this.postButton"> </formio>
`

As you can see I'm trying to get any event to trigger, I've tried with an without colons and this.

Form object button:
`
components: Array(2)
0: {input: true, label: "", key: "k1065", defaultValue: "1", type: "htmlelement", …}
1:
action: "custom"
custom: "postButton('eventOverride(Input_Event_0)');"
defaultValue: true
input: true
key: "eventOverride(Input_Event_0)"
label: "Trigger"
theme: "success"
type: "button"
__proto__: Object
length: 2
__proto__: Array(0)
key: "Input_Event_0"
theme: "default"
title: "Input_Event_0"
type: "panel"`

Note this bit `action: "custom" custom: "postButton('eventOverride(Input_Event_0)');"`

At the end of the day, I'd like it to call postMessage in my component.

It will call postMessage in a little bit of script I placed in the HTML, but then it's a mess trying to find the component again from globals and it trashes the modularity of it.

In short, I can get onChange events to work but no other v-on events.

I've lost days on this.  Help please!

**Comment by GabrielGoldsteinAnidea** [comment]
Also, using vue-formio 4.0., vue 2.6.12.

this.formio.events.onAny() does not get called when I click the button, let alone ever get to the 'customEvent' emitter.

**Comment by jeriah-formio** [comment]
We're currently addressing a backlog of GitHub issues. Closing this thread as it is outdated. Please re-open if it is still relevant. Thank you for your contribution!
